scripts/tutorial.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# **************************************************************************
+# *
+# * Authors:     J.M. De la Rosa Trevin (jmdelarosa@cnb.csic.es)
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
```

Comment:
I started updating to GPLv3...although not sure about the differences.

Reply:
We need agreement of all authors to officially update to gpl3 :)

Reply:
The main difference is that gpl3 forbids tivoization

Reply:
Uppss...my fault that I was changing it so happily :(
Are you OK with gplv3? Btw, what is tivoization?

Reply:
I just googled about it.

Reply:
I don't think it might be an issue for this project.